bin/main

```diff
@@ -1,52 +1,79 @@
 #!/usr/bin/env ruby
+require_relative '../lib/board'
+require_relative '../lib/player'
 
-posible_wins = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
-player_one_choices = []
-player_two_choices = []
-game_on = true
+# determine if an array includes given elements.
+class Array
+  def contain_all?(arr)
+    arr.all? { |elem| include?(elem) }
+  end
+end
```

Comment:
- [ ] No game logic should exist on main.rb, kindly move this block into the lib folder.